Added prerendering with traffic data example (#131)
moovweb-docs/xdn-docs: guides/static_prerendering.md

})
```

## Prerendering with traffic data

> XDN allows to prerender based on site traffic. This will ensure that most popular pages are always prerendered.

Reviewer: ```suggestion
The XDN can choose which pages to prerender based on site traffic, ensuring most popular pages are always prerendered.
```

> ### Example: Prerender 100 most requested pages

Reviewer: ```suggestion
### Example: Prerender the 100 most frequently viewed pages
```

```js
const { Router } = require('@xdn/core/router')

> module.exports = new Router().prerender([{ top: 100, headers: { foo: 'bar' } }])

Reviewer: You need to expand on this.  What's the headers config mean?  What other configs are possible here?

})
```

## Prerendering with traffic data

> The XDN can choose which pages to prerender based on site traffic, ensuring most popular pages are always prerendered.

Reviewer: ```suggestion
The XDN can choose which pages to prerender based on site traffic, ensuring the most popular pages are always available in the edge cache.
```

### Example: Basic Usage

```js
const { Router } = require('@xdn/core/router')

router = new Router().prerender([
  {

>     // How many pages should be prerendered based on site traffic and current configuration.

Reviewer: ```suggestion
    // the maximum number of pages that should be prerendered based on site traffic.
```

    top: 50,
  },
])
```

### Example: With cache splitting

```js
router = new Router().prerender([
  // Prerender with language cookie
  {
    top: 10,
    // Request headers that will be passed to your prerender request.
    // If you're splitting the cache by cookies or headers you can provide them

>     // using headers option

Reviewer: ```suggestion
    // using headers option.
```
